py/visdom/__init__.py

```diff
@@ -2209,3 +2209,75 @@ def mesh(self, X, Y=None, win=None, env=None, opts=None):
             'layout': _opts2layout(opts),
             'opts': opts,
         })
+    @pytorch_wrap
+    def double_yaxis_lines(self,X=None,Y1=None,Y2=None,opts=None,win=None,env=None):
+        '''
+        This function will create a line plot using plotly with different Y-Axis.
+        `X`  = A numpy array of the range.
+        `Y1` = A numpy array of the same count as X.
+        `Y2` = A numpy array of the same count as X.
+        Here are the `opts` options:
+        - `opts.height` : Height of the plot
+        - `opts.width` :  Width of the plot
+        - `opts.nameY1` : Axis name for Y1 plot
+        - `opts.nameY2` : Axis name for Y2 plot
+        - `opts.title` :  Title of the plot
+        - `opts.color_title_Y2` :  Color of the axis Title
+        - `opts.color_tick_Y2` :  Color of the axis Ticks
```

Comment:
nit: opts keys should be `lower_case_with_underscore`, even when referring to `Y1` and `Y2`.

Reply:
Changes Done!.